### DISCUSSION TITLE
New on Github

### DISCUSSION THREAD

**Discussion by smoriginals** [argyleink/open-props#510] [Q&A]
**I'm new on github and I don't know what is npm? People show is like **npm** version 1.5.7... something like that.**

**Comment by argyleink** [comment]
NPM is a package manager and public space for open source code, like these CSS variables =) it tries to make it easy to use the code while also getting updates as the packages on it are updated. 1.5.7 and stuff refer to the version of a package, giving you a sense of how up to date the version you're using is vs the one that's latest and published/public.

that help!?
